download/releaseNotes/releaseNotes8.adoc

```diff
@@ -46,7 +46,7 @@ Other noteworthy changes done during the migration to Quarkus:
 
 * OptaWeb Vehicle Routing back end has a new RESTful API.
   Client-server communication, that was previously done using WebSockets, now uses a combination of REST calls and Server-Sent Events.
-* OptaWeb Employee Rostering now uses Constraint Streams instead of DRL for score calculation.
+* OptaWeb Employee Rostering now uses ConstraintStreams instead of DRL for score calculation.
```

Comment:
Really? I've never once written it like this. This is not how humans write names.
It's `ConstraintStreams` as much as it's `OptaWebVehicleRouting`.

Reply:
@triceo I think the same but if you grep the source code in `optaplanner` repo, you'll see that `ConstraintStreams` occurs quite often. This should be unified when you and @ge0ffrey reach an agreement.

For now, I'll stick with `Constraint Streams` as this is almost consistently used here in `optaplanner-website`.

Reply:
In code, I kinda understand it, even though I would still prefer to only use "ConstraintStreams" if were actually referencing a class name somewhere.

In public-facing docs, I do not see any justification for "ConstraintStreams" whatsoever.